src/Files.App/MainWindow.xaml.cs

```diff
@@ -46,6 +48,9 @@ private void EnsureEarlyWindow()
             // Set title
             AppWindow.Title = "Files";
 
+            // Set icon
+            AppWindow.SetIcon(Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles/StoreLogo.ico"));
```

Comment:
```suggestion
            AppWindow.SetIcon(Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles/Dev/Logo.ico"));
```
There should be a similar logo in the `Release` and `Preview` folder.

Reply:
The build pipeline will automatically change the path in the code when it builds the release/preview version of Files.